easybuild/easyconfigs/l/libyuv/libyuv-20250227-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'CMakeMake'
+
+name = 'libyuv'
+version = '20250227'
+local_commit = 'c060118bea3f28ceb837d3c85e479d3bb4c21726'
+
+homepage = 'https://github.com/lemenkov/libyuv'
+description = """libyuv is an open source project that includes YUV scaling and conversion functionality."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['https://github.com/lemenkov/libyuv/archive/']
+sources = [{
+    "download_filename": "%s.tar.gz" % local_commit,
+    "filename": SOURCE_TAR_GZ,
+}]
+checksums = ['b509e836fde118b70f5f65b0c6835c724b37b63d4a5d52c2b49f76944645e948']
+
+builddependencies = [
+    ('CMake', '3.29.3'),
+    ('binutils', '2.42'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/yuvconvert', f'lib/{name}.so', f'lib/{name}.a', f'include/{name}.h'],
```

Comment:
This should be "lib/%%(name)s.%s" % SHLIB_EXT" and lib/%(name)s.a" etc to follow the style of EB